aspnetcore/blazor/globalization-localization.md

```diff
@@ -77,7 +77,10 @@ For current browser support of the preceding types, see [Can I use](https://cani
 
 Blazor WebAssembly uses a reduced globalization API and set of built-in International Components for Unicode (ICU) locales. For more information, see [.NET globalization and ICU: ICU on WebAssembly](/dotnet/core/extensions/globalization-icu#icu-on-webassembly).
 
-To load a custom ICU data file to control the app's locales, see [WASM Globalization Icu](https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-icu-wasm.md). Currently, manually building the custom ICU data file is required. .NET tooling to ease the process of creating the file is planned for a future .NET 8.0 preview release.
+<!-- UPDATE 8.0 Confirm that this remark is correct 
```

Comment:
@lewing can you please confirm the below is the current state in .NET 8?

Reply:
I believe it is, @ilonatommy can confirm

Reply:
@ilonatommy ... Here's the remark so that you don't need to go search the DIFF for it (sorry, @lewing that I didn't place this sooner!) ...

> To load a custom ICU data file to control the app's locales, see \[WASM Globalization Icu](https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-icu-wasm.md). Currently, manually building the custom ICU data file is required. .NET tooling to ease the process of creating the file is planned for .NET 9 in November, 2024.